benchmarks/routines/quantization.py

```
@@ -0,0 +1,825 @@
+    m = args.m
+    k = args.k
+    is_sf_swizzled_layout = args.is_sf_swizzled_layout
+    alignment = args.alignment
+    enable_pdl = args.enable_pdl
+    is_cuda_graph_compatible = not args.no_cuda_graph
+    run_refcheck = args.refcheck
+    res = []
+
+    # Validate k is divisible by 32 (sf_vec_size)
+    if k % 32 != 0:
+        raise ValueError(f"k ({k}) must be divisible by 32 (sf_vec_size)")
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)